docs/addons.md

```diff
@@ -48,6 +48,9 @@ spec:
 
 Read more about cluster autoscaler in the [official documentation](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler).
 
+##### Expander strategies
+The cluster autoscaler supports five different [expander strategies](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-are-expanders). The `price` expander is only supported on GCE. The `priority` expander requires additional configuration in a ConfigMap as described in [its documentation](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/expander/priority/readme.md) - please ensure that you have deployed this ConfigMap before enabling the `priority` expander.
```

Comment:
```suggestion
Cluster autoscaler supports different [expander strategies](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-are-expanders).

Please note that the "priority" expander requires additional configuration through a ConfigMap as described in [its documentation](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/expander/priority/readme.md). The expander requires that the the ConfigMap exists before being enabled.
```

Reply:
I'm puzzled about this change - it looks like you removed the note about the `price` expander only being supported on GCE. Wouldn't we want to have a note about this in the docs if we're going to validate it in the code? 🤔

Reply:
We don't need to duplicate CAS docs where things are graceful. Users can't "harm" themselves if they tried to use price on AWS.
Starting CAS with priority expander without the configmap would be "harmful" though.

Reply:
OK, I understand the distinction you are drawing, and I agree that it is better not to duplicate the autoscaler's docs unnecessarily. I reworded the docs in my latest commit.